src/components/tx/NonceForm/index.tsx

```diff
@@ -98,61 +95,59 @@ const NonceForm = ({ name, nonce, recommendedNonce, readonly }: NonceFormProps):
   }
 
   return (
-    <Autocomplete
-      value={value}
-      freeSolo
-      // On option select or free text entry
-      onInputChange={(_, value) => {
-        onChange(value ? Number(value) : '')
-      }}
-      options={options}
-      disabled={nonce == null || readonly}
-      getOptionLabel={(option) => option.toString()}
-      renderOption={(props, option) => <NonceFormOption nonce={option} {...props} />}
-      disableClearable
-      onClose={() => {
-        setPopper({ anchorEl: null, open: false })
-      }}
-      onOpen={(e) => {
-        setPopper({ anchorEl: e.currentTarget, open: true })
-      }}
-      PopperComponent={(props) => (
-        <Backdrop open={popper.open}>
-          <Popper anchorEl={popper.anchorEl} {...props} />
-        </Backdrop>
-      )}
-      renderInput={(params) => (
-        <TextField
-          {...params}
-          name={name}
-          onBlur={onBlur}
-          inputRef={ref}
-          type="number"
-          autoComplete="off"
-          error={!!fieldState.error}
-          label={label}
-          InputProps={{
-            ...params.InputProps,
-            endAdornment: !readonly &&
-              recommendedNonce !== undefined &&
-              recommendedNonce !== params.inputProps.value && (
-                <InputAdornment position="end">
-                  <Tooltip title="Reset to recommended nonce">
-                    <IconButton onClick={onResetNonce} size="small" color="primary">
-                      <RotateLeftIcon />
-                    </IconButton>
-                  </Tooltip>
-                </InputAdornment>
-              ),
-            readOnly: readonly,
-          }}
-          InputLabelProps={{
-            ...params.InputLabelProps,
-            shrink: true,
-          }}
-        />
-      )}
-    />
+    <>
+      <Backdrop open={backdrop} />
```

Comment:
An interesting workaround, but have you figured out why the flicker happens specifically with this dropdown?

Reply:
It's related to the gas price polling [here](https://github.com/safe-global/web-core/blob/dev/src/hooks/useGasPrice.ts#L64-L78). The `Backdrop` seems to be very tempermental between renders.

Reply:
Sounds like the polling re-renders more than necessary in that form. I would take another look at that.

Reply:
I deleted the backdrop entirely. It now looks the same as the AddressBookInput. Just a dropdown.

Reply:
Without the `Backdrop`, the options blur into the background though. It was suggested [here](https://github.com/safe-global/web-core/pull/1079#issuecomment-1327121065):

![image](https://user-images.githubusercontent.com/20442784/205278136-667f757e-1bce-4c5b-8312-5d92c43d6c9d.png)

cc @usame-algan